i18n/en.json

```diff
@@ -204,7 +204,7 @@
 	"smw-ask-sorting": "Sorting",
 	"smw-ask-search": "Search",
 	"smw-ask-format-selection-help": "For a detailed description, please visit the $1 help page.",
-	"smw-ask-query-search-info":"The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|<code>$2</code> (from cache)|<code>$2</code>}} in $4 {{PLURAL:$4|second|seconds}}.",
+	"smw-ask-query-search-info":"The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|<code>$2</code>|<code>$2</code> (from cache)}} in $4 {{PLURAL:$4|second|seconds}}.",
```

Comment:
If you have `{{PLURAL:$1|1=something|other}}` it basically means that language specific plural rules are ignored: first parameter is only used for number 1 (and not whatever the language considers singular), and the other parameter for everything else. I don't mind which part is `something` and which part `other`.